src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,5 @@
+PORT=8080
```

Comment:
do you set somewhere 'dev' profile? as soon as i understand it's not used at all since u set all the variables in docker-compose file. am i right?

Reply:
i mean the whole file. don't know how to make a comment on it